lib/workflow/stores/mongo.js

```diff
@@ -283,8 +284,15 @@ function mongoStoreFactory(waterline, Promise, Constants, Errors, assert, _) {
                 instanceId: 1
             }
         };
+        var promise = waterline.graphobjects.findAndModifyMongo(query, {}, graph, options);
 
-        return waterline.graphobjects.findAndModifyMongo(query, {}, graph, options);
+        if (isNew) {
+            promise.then(function () {
+                waterline.graphobjects.publishRecord('created', graph);
```

Comment:
Why need the extra publish? As I see the `created` event has been published in model's `afterCreate`.